content/en/apps/features/integrations/android.md

```diff
@@ -17,6 +17,84 @@ When the user clicks on a link to a CHT instance from an SMS, email, WhatsApp, o
 
 Users can choose "Always" to skip this prompt in future. The prompt may look different depending on the version of Android being used.
 
+## Android App Links verification
+
+Starting with Android 12, Android supports associating an app with a domain and automatically verify this association
+to allow deep links to immediately open the app without requiring your user to select the app.
+To get this working, you need to host a Digital Asset Links JSON file at `https://<domain.name>/.well-known/assetlinks.json`
+containing some information about your app to associate it with your domain.  
+More information available on the [official Android docs](https://developer.android.com/training/app-links/verify-android-applinks).
+
+### Hosting `assetlinks.json` with the CHT
+
+Since CHT Core version 4.7.0, the CHT supports serving `assetlinks.json` by adding it to your app settings.
+All you have to do to make the CHT serving your assetlinks at `/.well-known/assetlinks.json` is to:
+1. Define your assetlinks in either `base_settings.json` or `app_settings/assetlinks.json`
+2. Compile and upload your app settings using cht-conf `cht-conf compile-app-settings upload-app-settings`
+
+You can read more about the structure and contents of assetlinks in [the assetlinks configuration docs]({{< ref "apps/reference/app-settings/assetlinks" >}}).
```

Comment:
question: This is not showing up as a clickable link. Afer this PR is merged and published to prod, does this show up?

Reply:
I'm not sure could be happening here :raised_eyebrow:  the link is showing up and working fine locally 
![image](https://github.com/medic/cht-docs/assets/13026820/146f37ec-05d9-43fb-9b53-9cf5447650b6)

Reply:
I see. Then, it should be working fine because I'm viewing it from the markdown preview of Github.